src/shared/tpm2-util.c

```diff
@@ -1117,7 +1117,11 @@ static int tpm2_make_policy_session(
                                 ESYS_TR_NONE,
                                 NULL,
                                 &pubkey_tpm2,
+#if HAVE_TSS2_ESYS3
+                                ESYS_TR_RH_OWNER,
+#else
                                 TPM2_RH_OWNER,
```

Comment:
can you add a brief comment here, explaining why we have this here i.e. `/* on old tpm2-tss there's a bug: …` and something useful?

Reply:
(or in other words: move a superbrief summary of the commit msg into the code here as a comment)

Reply:
No problem, pushed!